src/components/Countries.js

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowCircleRight } from '@fortawesome/free-solid-svg-icons';
+import data from './data';
+import '../assets/country.css';
+
+function Countries() {
+  const [info, setInfo] = useState([]);
+  const [searchInput, setSearchInput] = useState('');
+  const [filteredInfo, setFilteredInfo] = useState([]);
+
+  useEffect(() => {
+    setInfo(data);
+    setFilteredInfo(data);
+  }, []);
+
+  const handleSearchInputChange = (event) => {
+    const inputValue = event.target.value;
+    setSearchInput(inputValue);
+
+    const filteredCountries = info.filter((country) => country.country.toLowerCase()
+      .includes(inputValue.toLowerCase()));
+    setFilteredInfo(filteredCountries);
+  };
+  /* eslint-disable */ 
+  return (
```

Comment:
- [ ] Let's try to not disable Linter checks in our codebase. The Linter cheks are there to help us write structured and professional code and disabling them defies the purpose of setting it up. Kindly work on the Linter errors flagging 👍🏽